src/apis/staked-relayer.ts

```diff
@@ -24,6 +24,12 @@ export interface StakedRelayerAPI {
     getAllActiveStatusUpdates(): Promise<Array<{ id: u256; statusUpdate: StatusUpdate }>>;
     getAllInactiveStatusUpdates(): Promise<Array<{ id: u256; statusUpdate: StatusUpdate }>>;
     getAllStatusUpdates(): Promise<Array<{ id: u256; statusUpdate: StatusUpdate }>>;
+    getFees(stakedRelayerId: AccountId): Promise<PolkaBTC>;
+    getSLA(stakedRelayerId: AccountId): Promise<number>;
+    getScore(stakedRelayerId: AccountId): Promise<string>;
+    getTotalScore(): Promise<string>;
```

Comment:
Not sure what the total score would be? The score among all relayers? If yes, then we don't really need this.

Reply:
See the screenshot attached